modules/statics/src/coins.ts

```diff
@@ -186,6 +186,11 @@ const POLYGON_FEATURES = [
 ];
 const POL_FEATURES = [
   ...AccountCoin.DEFAULT_FEATURES,
+  CoinFeature.STAKING,
+  CoinFeature.MULTISIG_COLD,
+  CoinFeature.METAMASK_INSTITUTIONAL,
```

Comment:
let's remove `CoinFeature.METAMASK_INSTITUTIONAL` - don't think it supports that